src/measure_theory/measurable_space.lean

```diff
@@ -797,6 +796,14 @@ def set.singleton [measurable_space α] (a:α) : measurable_equiv ({a} : set α)
   measurable_to_fun := measurable_const,
   measurable_inv_fun := measurable_const }
 
+lemma set.image_aux1 {f : α → β} (hf : function.injective f) (u s : set α) :
+  {x : f '' s | ((equiv.set.image f s hf).inv_fun x).val ∈ u} = subtype.val ⁻¹' (f '' u) :=
```

Comment:
This looks like a lemma that should go to `data.equiv.basic`. And AFAIR `simp`-normal forms are `equiv.symm` and `coe`, not `equiv.inv_fun` and `subtype.val`.

Reply:
Maybe. I don't even know what it was really saying, and don't know a good name for it, or whether it will be useful again.